.github/workflows/ci.yml

```diff
@@ -45,6 +45,19 @@ jobs:
         shell: bash
         run: sudo make install
 
+      - name: Setup Elixir Host SDK
+        uses: actions/setup-elixir@v1
+        with:
+          experimental-otp: true
+          otp-version: '25.0'
+          elixir-version: '1.14.0'
+
+      - name: Test Elixir Host SDK
+        run: |
+          cd elixir
+          mix deps.get
+          MIX_ENV=test mix do compile, test
+
```

Comment:
Works on my machine but can't seem to find the `.so` on github actions.

Reply:
Hm, yea this works locally for me as well. I can look into it further later this weekend!